modules/pel/peltool/peltool.py

```diff
@@ -228,6 +228,9 @@ def parsePEL(stream: DataStream, config: Config, exit_on_error: bool):
     if config.non_serviceable_only and uh.isServiceable():
         return "", ""
     
+    if config.passed_severity != "0" and uh.eventSeverity != int(config.passed_severity, 16):
```

Comment:
I suggest we should have a utility API to implement these rules, which will determine whether the PEL needs to be considered as a return if the rules are met. This will prevent us from duplicating the same rules in multiple locations.

We can introduce this in a separate PR once we've finalized the expected behavior for -a, -l, and -n, etc.

Reply:
Thank you for pushing https://github.com/ibm-openbmc/openpower-pel-parsers/pull/55. I am unable to mark this comment as resolved, so please close it.